poetry/core/masonry/builders/builder.py

```diff
@@ -364,17 +363,12 @@ def convert_author(cls, author: str) -> Dict[str, str]:
     @classmethod
     @contextmanager
     def temporary_directory(cls, *args: Any, **kwargs: Any) -> ContextManager[str]:
```

Comment:
This is an example of a now-unnecessary wrapper function, and I would prefer to leave it in place on the first commit and then excise it entirely on a follow-up commit